sdk/cognitivelanguage/azure-ai-language-questionanswering/setup.py

```diff
@@ -61,13 +61,14 @@
         # This means any folder structure that only consists of a __init__.py. 
         # For example, for storage, this would mean adding 'azure.storage' 
         # in addition to the default 'azure' that is seen here.
-        'azure'
+        'azure',
         'azure.ai',
         'azure.ai.language',
     ]),
     install_requires=[
         'azure-core<2.0.0,>=1.16.0',
-        "msrest>=0.6.21"
+        'msrest>=0.6.21',
+        'chardet>=3.0.2,<5'
```

Comment:
Nit: trailing comma makes the diff cleaner next time, like you did on line 66 above.